<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyContacts</class>
 <widget class="QMainWindow" name="EasyContacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy Contacts</string>
  </property>
  <property name="windowIcon">
   <iconset theme="contact-new"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_pretraga"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_uredi">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Uredi</string>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_obrisati">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Obrisati</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuO_aplikaciji">
    <property name="title">
     <string>Pomoć</string>
    </property>
    <addaction name="actionO_programu"/>
    <addaction name="actionDokumentacija"/>
    <addaction name="separator"/>
    <addaction name="actionGeneri_i_Demo_podatke"/>
   </widget>
   <widget class="QMenu" name="menuKontakti">
    <property name="title">
     <string>Kontakti</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIzradi_novi_kontakt"/>
    <addaction name="separator"/>
    <addaction name="actionUvoz"/>
    <addaction name="actionIzvoz"/>
   </widget>
   <addaction name="menuKontakti"/>
   <addaction name="menuO_aplikaciji"/>
  </widget>
  <action name="actionUvoz">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Uvoz</string>
   </property>
  </action>
  <action name="actionIzvoz">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>Izvoz</string>
   </property>
  </action>
  <action name="actionIzradi_novi_kontakt">
   <property name="icon">
    <iconset theme="contact-new"/>
   </property>
   <property name="text">
    <string>Izradi novi kontakt</string>
   </property>
  </action>
  <action name="actionGeneri_i_Demo_podatke">
   <property name="text">
    <string>Generiši Demo podatke</string>
   </property>
  </action>
  <action name="actionO_programu">
   <property name="text">
    <string>O programu</string>
   </property>
  </action>
  <action name="actionDokumentacija">
   <property name="text">
    <string>Dokumentacija</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
